Core.ApplicationServices/SystemUsage/Migration/ItSystemUsageMigrationService.cs

```diff
@@ -252,57 +229,30 @@ public bool CanExecuteMigration()
             return _authorizationContext.HasPermission(new SystemUsageMigrationPermission());
         }
 
-        private bool DeleteExhibits(ItSystemUsageMigration migration)
+        private bool DeleteUsedByRelationInterfaces(ItSystemUsageMigration migration)
```

Comment:
Naming er skør her for der er intet i din datamodel der retfærdiggører det navn. 

Omdøb til: PerformRelationMigrations